portlets/src/main/webapp/vue-app/rules/components/drawers/RuleFormAutomaticFlow.vue

```diff
@@ -261,7 +261,7 @@ export default {
       this.loading = true;
       return this.$gamificationConnectorService.getConnectors(eXo.env.portal.userName)
         .then(connectors => {
-          const enabledConnectors = connectors?.filter(connector => connector.enabled) || [];
+          const enabledConnectors = connectors?.filter(connector => connector.enabled || (eXo.env.portal.EvmConnectorEnabled && connector.name === 'evm')) || [];
```

Comment:
IMO it must be handled in the back office (connector plugin) . We need to maintain the generality of this component